docs/guides/AutomateHEXFileBuilding.rst

```diff
@@ -3,36 +3,37 @@
+
+Continuous delivery requires automation of the process that builds the HEX file of an application.
 
 .. figure:: ./images/github-release-with-hex-files.png
    
    GitHub release with attached HEX files
 
-One precondition for that is the automation of the process that builds the HEX file of an application.
-
-Getting all these steps right takes a while, because configuring NCS and Zephyr includes `many <https://developer.nordicsemi.com/nRF_Connect_SDK/doc/1.4.1/nrf/gs_installing.html>`_ `individual <https://developer.nordicsemi.com/nRF_Connect_SDK/doc/1.4.1/zephyr/getting_started/installation_linux.html>`_ tasks, but afterwards you have a reproducible recipe to build your HEX files, as part of your source code project.
+Getting all the installation and configuration steps right takes a while, but afterwards you have a reproducible recipe to build your HEX files, as part of your source code project.
 
 Not only does that mean you can provide up-to-date HEX files with zero effort, but using a CI runner allows you to automate your software releases using a project called `semantic release <https://github.com/semantic-release/semantic-release>`_.
 
 .. note::
 
-    By default ``semantic-release`` will get the ``reposityUrl`` it uses to  determine the changes between releases from the ``package.json`` which after forking the ``firmware`` repo will still point to ``https://github.com/NordicSemiconductor/asset-tracker-cloud-firmware-aws.git``. 
-    Either update that to the URL of your fork, or `provide it as an argument <https://semantic-release.gitbook.io/semantic-release/usage/configuration#repositoryurl>`_ to the ``semantic-release`` CLI.
+
+    By default, the semantic-release gets the ``reposityUrl`` from the :file:`package.json` file. It uses the URL to determine the changes between releases.
+    After forking the ``firmware`` repository, the URL will still point to ``https://github.com/NordicSemiconductor/asset-tracker-cloud-firmware.git``.
+    Either update that to the URL of your fork, or `provide it as an argument <https://semantic-release.gitbook.io/semantic-release/usage/configuration#repositoryurl>`_ to the semantic-release CLI.
 
 The `workflow definition for GitHub Actions <https://github.com/NordicSemiconductor/asset-tracker-cloud-firmware-aws/blob/saga/.github/workflows/build-and-release.yaml>`_  automatically builds HEX files and attaches them to the GitHub release.
 
-Following this example, all developers need to do is to format their commit messages following a certain schema (for more information see :ref:`guides-versionining-how-to-release-a-new-version-of-a-package`) to trigger a new release.
+Following this example, just format your commit messages following a certain schema (for more information see :ref:`guides-versionining-how-to-release-a-new-version-of-a-package`) to trigger a new release.
```

Comment:
Please avoid language that paints tasks in a simplifying manner, like "just", "simply" (so called *lazy words*), because it might seem trivial to you, and you can "just do it", but others may not have the same level of knowledge.
See also https://css-tricks.com/words-avoid-educational-writing/